Source/WebCore/svg/SVGSVGElement.cpp

```diff
@@ -139,6 +139,17 @@ void SVGSVGElement::setCurrentTranslate(const FloatPoint& translation)
 
 void SVGSVGElement::updateCurrentTranslate()
 {
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
+    if (document().settings().layerBasedSVGEngineEnabled()) {
+        if (auto* renderer = this->renderer()) {
```

Comment:
What should we do if there is no renderer? Could it be tested first thing and return if null? Right now it seems to be handled a bit inconsistent between legacy and LBSE.

Reply:
It apears so, the updateSVG... method below doesn't do anything if renderer is null. However your suggestion is still valid, to avoid that confusion I'll simply move the null check for both legacy/LBSE on top.